components/EditMenu.tsx

```
@@ -0,0 +1,180 @@
+
+	if (albumTitle === null) return null;
+
+	return (
+		<div className="relative z-30">
+			<button
+				onClick={toggleMenu}
+				className="flex items-center justify-center w-10 h-10 rounded-full bg-orange-500 text-white hover:bg-orange-600 transition-colors shadow-sm"
+				aria-label={isMenuOpen ? "メニューを閉じる" : "編集メニューを開く"}
+			>
+				{isMenuOpen ? <FiX size={20} /> : <FiSettings size={20} />}
+			</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)